intranet/static/js/eighth/statistics.js

```diff
@@ -0,0 +1,66 @@
+/* global $ */
+$(document).ready(function() {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Function 'anonymous' has a complexity of 5.](https://www.codacy.com/app/tjcsl/ion/file/2620617210/issues/source?bid=3176160&fileBranchId=3299251#l2)